database/src/main/java/info/metadude/android/eventfahrplan/database/sqliteopenhelper/SessionsDBOpenHelper.java

```diff
@@ -10,18 +10,18 @@
 import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionByNotificationIdTable;
 import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable;
 import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable.Columns;
-import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable.Defaults;
-import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable.Values;
 
 public class SessionsDBOpenHelper extends SQLiteOpenHelper {
 
-    private static final int DATABASE_VERSION = 10;
+    private static final int DATABASE_VERSION = 1;
 
-    private static final String DATABASE_NAME = "lectures"; // Keep table name to avoid database migration.
+    private static final String DATABASE_NAME = "sessions";
 
     private static final String SESSIONS_TABLE_CREATE =
             "CREATE TABLE " + SessionsTable.NAME + " (" +
+                    BaseColumns._ID + " INTEGER PRIMARY KEY AUTOINCREMENT," +
                     Columns.SESSION_ID + " TEXT, " +
```

Comment:
We should use this opportunity to remove the `SESSION_ID` column from the database.